uavcan/equipment/gnss/ECEFPosition.uavcan

```diff
@@ -0,0 +1,17 @@
+#
+# Nested type.
+# GNSS ECEF status field.
+#
+# ECEF is an acronym for Earth-Centered-Earth-Fixed, which is a cartesian
+# coordinate system which rotates with the earth. The origin (0,0,0) is
+# located at the center of the earth. The x-axis is a vector pointing from
+# the origin with positive direction towards 0 degrees latitude and
+# longitude (equator, at the prime meridian). The z-axis is a vector
+# pointing from the origin towards the north-pole. The y-axis completes a
+# right-handed coordinate system.
+
+int34 x_mm                         # X-axis coordinate in mm
+int34 y_mm                         # Y-axis coordinate in mm
+int34 z_mm                         # Y-axis coordinate in mm
```

Comment:
We could sacrifice one more byte to make the message definition suitable for use in space. This is what I propose:

```
int36[3] xyz_mm
void4
```